mne/report.py

```diff
@@ -551,7 +551,7 @@ def _build_html_slider(slices_range, slides_klass, slider_id):
 footer_template = HTMLTemplate(u"""
 </div></body>
 <div class="footer">
-        &copy; Copyright 2012-2013, MNE Developers.
+        &copy; Copyright 2012-{{current_year}}, MNE Developers.
```

Comment:
my fault, cherry-picked the first commit, manually did the second and forgot to extract all relevant lines.